app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -46,6 +47,10 @@ def ems_inv_to_hashes
 
       private
 
+      def get_network_interfaces
+        @data[:network_interfaces] = gather_data_for_this_region(@nis)
```

Comment:
@bzwei Since I made another change, I went ahead and refactored gather_data_for_this_region as well.